.ci-self/build-platform.yml

```diff
@@ -68,3 +68,19 @@ jobs:
       #   inputs:
       #     PathtoPublish: "_release"
       #     ArtifactName: ${{ parameters.platform }}
+  - job: "Debug ${{ parameters.platform }}"
+    condition: failed() # this will only if the main job fails
+    steps:
+      - task: PublishBuildArtifacts@1
+      pool:
+        vmImage: ${{ parameters.vmImage }}
+        demands: node.js  
+      displayName: 'Upload project directory'
+      # TODO: The CI Build caches are pulled down by the last successful buildID
```

Comment:
I don't think this will make it a success so should be fine

Reply:
I'm experimenting with something - creating downloadable project snapshots that can be locally inspected. Not sure what it will turn out to be, but lets see

Reply:
So, it must only run if the main job fails